backend/routers/items.py

```
@@ -199,6 +199,48 @@ async def get_categories():
+    today = date.today()
+    
+    # 在庫切れ予定日とnotify_days_beforeが設定されている商品を取得
+    items = db.query(Item).filter(
+        Item.user_id == current_user.id,
+        Item.oos_date.isnot(None),
+        Item.notify_days_before.isnot(None)
+    ).all()
+    
+    upcoming_items = []
+    for item in items:
+        # 通知予定日を計算（在庫切れ予定日 - 通知日数）
```

Ruff (0.14.4)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)